supplemental_notebooks/2022-09-20_muddiest_points.Rmd

```diff
@@ -0,0 +1,163 @@
+When quantifying gene expression in the dataset, all transcripts that have the same UMI are actually an artifact of PCR amplification, because during initial capture, no two RNA molecules should have the same UMI. This means that any transcripts with the same UMI should be counted only as a single transcript because they are merely just PCR copies of the same original transcript and do not accurately reflect gene expression. 
+To quantify gene expression of a particular gene in a given cell we count up the total number transcripts that map to that gene and contain unique UMIs, discarding any duplicate UMIs as they are artifacts of PCR amplification. 
+The final value that you see in the counts matrix will correspond to that total number of unique UMIs, after duplicates have removed. 
+
+Let's read in the filtered data saved after performing filtering of the the bladder sample in `02-filtering_scRNA.Rmd` and take a closer look at the counts data:
+
+```{r}
+# file paths to filtered sce 
+data_dir <- file.path("data", "tabula-muris")
+filtered_sce_file <- file.path(data_dir, "filtered", "filtered_sce.rds")
+
+# read in filtered sce 
+filtered_bladder_sce <- readr::read_rds(filtered_sce_file)
+```
+
+
+```{r}
+counts(filtered_bladder_sce)[1:5, 1:5]
+```
+
+Here we are looking at the first 5 columns and the first 5 rows of the counts matrix. 
+You can see that the names of the columns contain nucleotide strings, these are the cell barcodes. 
+This means that for any gene that has expression > 0 in that column, they had transcripts that were tagged with that same cell barcode and were from the same original cell. 
+
+We can also look at individual genes and identify how many copies of each of the genes were originally present in the cell, PRIOR to any PCR amplification. 
+For example, the gene ENSMUSG00000000001 had 2 copies present in all cells except for the 5th cell which shows 7 copies. 
+This means that there were 2 transcripts present in the cell that mapped to this gene that each were tagged with their own UMI, and therefore there were 2 UMIs associated with this gene.
+Any duplicates of that UMI that were present after amplification were removed prior to making this counts matrix as they were only an artifact of amplification. 
+
+
+## SingleCellExperiment objects 
+
+In the `02-filtering_scRNA.Rmd` notebook, we introduced the concept of using a `SingleCellExperiment` object to store the cell by gene counts matrix after running Alevin. 
+The reason we like to use an object like this, rather than simply store the gene by cell counts matrix as a matrix on its own, is because with a `SingleCellExperiment` object we can add additional information about each of the cell barcodes in the `colData` and each of the genes identified in our experiment in the `rowData`. 
```

Comment:
```suggestion
The reason we like to use an object like this, rather than simply store the gene-by-cell counts matrix as a matrix on its own, is because we can add additional information into a `SingleCellExperiment` object, such as the cell barcodes in the `colData` slot and each of the genes identified in our experiment in the `rowData` slot. 
```